src/main/cpp/UtilityTile.cpp

```diff
@@ -37,11 +41,7 @@ UtilityTile::getCostToBuy() const
 int 
 UtilityTile::getRent(int roll) const 
 {
-	//If there isn't an owner
-	if(!owner) {
-		return 0;
-	}
-	else if(owner->getNumUtilities() == 2) {
+	if(owner->getNumUtilities() == 2) {
```

Comment:
You might still want that null check in there, the owner could be null.

Reply:
It checks for it in the "landOnUtilityTile" and "landOnRailroadTile" functions in Monopoly.cpp. The "payRentTo" function is called, which then calls the "getRent" function. I'll put the "payRentTo" function in the "landOnHouseTile" function to save a few extra lines.